docs/contributing.rst

```diff
@@ -47,18 +47,18 @@ The Hydroshare team does our best to respond to pull requests within three busin
 
 Starting on Sept 1, 2017, CUAHSI will assume the role of the "benevolent dictator" of this production Hydroshare code base. After that point, pull requests into develop (and ultimately master) will only be merged by CUAHSI staff or other appointed staff from partner institutions. Other users can still:
```

Comment:
role of the "benevolent dictator" of this --> role of the "benevolent dictator" of the